pandas_sphinx_theme/static/css/theme.css

```diff
@@ -0,0 +1,387 @@
+/*
+ * nature.css_t
```

Comment:
This is the auto-generated file? 
If so, can we have a comment at the top saying this?

Reply:
A comment can be added saying it is autogenerated.